internal/audit/kafka/conf.go

```diff
@@ -35,6 +35,8 @@ type Conf struct {
 	ClientID string `yaml:"clientID" conf:",example=cerbos"`
 	// Brokers list to seed the Kafka client.
 	Brokers []string `yaml:"brokers" conf:"required,example=['localhost:9092']"`
+	// Compression sets the compression algorithm to use in order of priority. Valid values are "none", "gzip", "snappy","lz4", "zstd". Default is ["snappy", "none"].
+	Compression []string `yaml:"compression" conf:",example=['zstd', 'snappy', 'none']"`
```

Comment:
Due to a bug in the automated documentation generator (#1638) we have to slightly alter this example or it will produce invalid documentation. 
```suggestion
	Compression []string `yaml:"compression" conf:",example=['zstd']"`
```